### ISSUE TITLE
npm fails to install dependencies

### ISSUE THREAD

**Issue by mistersandman** [MetaMask/gaba#72]
`npm install gaba` fails to install on an empty nodejs project with

```
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git checkout e307359ca9ea6be4ded6ac58ac7e16f192ae4e2a
npm ERR! fatal: reference is not a tree: e307359ca9ea6be4ded6ac58ac7e16f192ae4e2a
```

the following dependency can not be resolved
https://github.com/MetaMask/gaba/blob/b0cbc49b5ccf311207a7e79fd57b420f722c0fc0/package.json#L76
it references an nonexistent commit in the respective repository

**Comment by whymarrh** [comment]
Fixed in #73

**Comment by whymarrh** [comment]
Thanks for the report!
